featurebyte/api/api_handler/feature_job_setting_analysis_list_handler.py

```diff
@@ -0,0 +1,27 @@
+"""
```

Comment:
Should we just remove `_list_handler` from the filename here?

Reply:
sure